datasize/src/std.rs

```diff
@@ -264,6 +305,19 @@ where
     }
 }
 
+impl<T> DataSize for std::cell::Cell<T>
+where
+    T: DataSize + Copy,
+{
+    const IS_DYNAMIC: bool = <T as DataSize>::IS_DYNAMIC;
+
+    const STATIC_HEAP_SIZE: usize = <T as DataSize>::STATIC_HEAP_SIZE;
+
+    fn estimate_heap_size(&self) -> usize {
+        self.get().estimate_heap_size()
```

Comment:
This looks like something that could be potentially painful, like a hidden performance killer. Do we really want this?

Reply:
Sounds fair, I omitted that this `get` made a copy.